zkevm-circuits/src/evm_circuit/util/math_gadget/mul_add_words.rs

```diff
@@ -80,7 +80,7 @@ impl<F: Field> MulAddWordsGadget<F> {
         let overflow = carry_hi_expr.clone()
             + a_limbs[1].clone() * b_limbs[3].clone()
             + a_limbs[2].clone() * b_limbs[2].clone()
-            + a_limbs[3].clone() * b_limbs[2].clone()
+            + a_limbs[3].clone() * b_limbs[1].clone()
```

Comment:
Finding this typo makes me feel this unit test has a little bit value :p

Reply:
Nice!